packages/expo-media-library/android/src/main/java/expo/modules/medialibrary/albums/migration/MigrateAlbum.kt

```diff
@@ -0,0 +1,55 @@
+package expo.modules.medialibrary.albums.migration
+
+import expo.modules.medialibrary.MediaLibraryUtils.AssetFile
+import android.os.AsyncTask
+import android.content.ContentValues
+import android.provider.MediaStore
+import android.content.ContentUris
+import android.content.Context
+import android.os.Build
+import androidx.annotation.RequiresApi
+import expo.modules.core.Promise
+import expo.modules.medialibrary.MediaLibraryConstants
+import expo.modules.medialibrary.MediaLibraryUtils
+import java.io.File
+
+@RequiresApi(Build.VERSION_CODES.R)
+internal class MigrateAlbum(
+  private val context: Context,
+  private val assetFiles: List<AssetFile>,
+  private val albumDirName: String,
+  private val promise: Promise
+) : AsyncTask<Void?, Void?, Void?>() {
+  override fun doInBackground(vararg voids: Void?): Void? {
```

Comment:
You can do something with that type to make it looks better. If you use a different type here (Unit or Any), you don't have to return null. However, I know, you're going to rewrite that using corutines. So leave it as it's.